lib/fastlane/actions/tryouts.rb

```diff
@@ -0,0 +1,124 @@
+                                     env_name: "TRYOUTS_APP_ID",
+                                     description: "Tryouts application hash",
+                                     verify_block: proc do |value|
+                                       raise "No application identifier for Tryouts given, pass using `app_identifier: 'application hash'`".red unless value and !value.empty?
+                                     end),
+          FastlaneCore::ConfigItem.new(key: :api_token,
+                                     env_name: "TRYOUTS_API_TOKEN",
+                                     description: "API Token for Tryouts Access",
+                                     verify_block: proc do |value|
+                                       raise "No API token for Tryouts given, pass using `api_token: 'token'`".red unless value and !value.empty?
+                                     end),
+          FastlaneCore::ConfigItem.new(key: :build_file,
+                                     env_name: "TRYOUTS_BUILD_FILE",
+                                     description: "Path to your IPA or APK file. Optional if you use the `gym` or `xcodebuild` action",
+                                     default_value: Actions.lane_context[SharedValues::IPA_OUTPUT_PATH],
+                                     verify_block: proc do |value|
+                                       raise "Couldn't find build file at path '#{value}'".red unless File.exist?(value)
+                                     end),
+          FastlaneCore::ConfigItem.new(key: :notes,
+                                     env_name: "TRYOUTS_NOTES",
+                                     description: "Release notes",
+                                     is_string: true,
+                                     optional: true),
+          FastlaneCore::ConfigItem.new(key: :notes_path,
+                                     env_name: "TRYOUTS_NOTES_PATH",
+                                     description: "Release notes text file path. Overrides the :notes paramether",
+                                     verify_block: proc do |value|
+                                       raise "Couldn't find notes file at path '#{value}'".red unless File.exist?(value)
+                                     end,
+                                     optional: true),
+          FastlaneCore::ConfigItem.new(key: :notify,
+                                     env_name: "TRYOUTS_NOTIFY",
+                                     description: "Notify testers? 1 for yes",
+                                     default_value: "1"),
+          FastlaneCore::ConfigItem.new(key: :status,
+                                     env_name: "TRYOUTS_STATUS",
+                                     description: "2 to make your release public. Release will be distributed to available testers. 1 to make your release private. Release won't be distributed to testers. This also prevents release from showing up for SDK update",
```

Comment:
You might want to add a `verify_block` for this to make sure no other number is passed